docs/commands/GetArchiveDaysMC.md

```diff
@@ -0,0 +1,131 @@
+# GetArchiveDaysMC
+
+Command to request/receive day pulse counter's values from device archive.
+
+
+## Request
+
+### Format
+
+| Size   | Type                                         | Field                                            |
+| ------ | -------------------------------------------- | ------------------------------------------------ |
+| `1`    | `byte`                                       | command id = `0x1b`                              |
+| `1`    | `byte`                                       | command size = `4+`                              |
+| `2`    | [packed date](../types.md#packed-date)       | [start date](#date)                              |
+| `1..5` | [extended value](../types.md#extended-value) | [channels bit set](../types.md#channels-bit-set) |
+| `1`    | `byte`                                       | [amount of days](#days)                          |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **date**
```

Comment:
it't best to match this name and the name in the Field column in Format

and for Response

Reply:
fixed